ResidentInformationApi.Tests/V1/Controllers/ResidentInformationControllerTests.cs

```diff
@@ -0,0 +1,30 @@
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using ResidentInformationApi.V1.Boundary.Requests;
+using ResidentInformationApi.V1.Controllers;
+
+namespace ResidentInformationApi.Tests.V1.Controllers
+{
+    [TestFixture]
+    public class ResidentInformationControllerTests
+    {
+        private ResidentInformationController _classUnderTest;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _classUnderTest = new ResidentInformationController();
+        }
+
+        [Test]
+        public void ListContactsReturnsAResponse()
```

Comment:
It would be good to see failure test cases as well if there are some.

Reply:
These have been added in the latest commits